### ISSUE TITLE
Fix issues with 'other' options

### ISSUE THREAD

**Issue by IAlferink** [digital-york/oasis#219]
There are three issues with the _other_ options in the metadata fields:
1. For the _Journal of publication_ and _Language feature_ fields no free text box appears
2. The _Country / region_ field does not have an _other_ option
3. For _Language feature_ the _other_ option should appear at the bottom of the list to be consistent with the other fields. It is currently listed under O

**Comment by IAlferink** [comment]
For Journal of publication the free text box doesn't appear.

**Comment by yf508** [comment]
I've tested both in Chrome & Firefox but cannot see any problem. Which browser did you use? @IAlferink 

![image](https://user-images.githubusercontent.com/437701/46603808-21070180-caec-11e8-9882-0895d10ee856.png)

**Comment by IAlferink** [comment]
@yf508 Safari again.

**Comment by IAlferink** [comment]
@yf508 
<img width="813" alt="screen shot 2018-10-08 at 14 34 53" src="https://user-images.githubusercontent.com/40735172/46612028-59680900-cb07-11e8-9daa-51090207f46f.png">

**Comment by yf508** [comment]
This issue may have the same cause as #241, due to the recent update of Safari on 17 Sep 2018.

**Comment by IAlferink** [comment]
Yes, could be.  I've tried with Chrome as well and that's fine for me.

**Comment by IAlferink** [comment]
@yf508 Related problem: When I tick the _other_ option for _Journal of publication_ the text box doesn't immediately appear. Weirdly, you have to click on something else and then back on the field for it to appear.

**Comment by paluchas** [comment]
I have retested this on my Safari Version 12.0.1 (13606.2.104.1.2). The other option works on other field with select boxes and scroll lists. However, it will not work for drop down list. If we not fix this we need to add contextual text for those fields informing user about this Safari issue.

**Comment by paluchas** [comment]
Related to #277 and #245

**Comment by paluchas** [comment]
Summary of the current status:
- Firefox works ok
- Chrome requires a double click on other filed in Journal List.
- Safari done't work
